docs/PAIMode.md

```diff
@@ -0,0 +1,79 @@
+**Run an Experiment on OpenPAI**
+===
+NNI supports running an experiment on [OpenPAI](https://github.com/Microsoft/pai) (aka pai), called pai mode. Before starting to use NNI pai mode, you should have an account to access an [OpenPAI](https://github.com/Microsoft/pai) cluster. See [here](https://github.com/Microsoft/pai#how-to-deploy) if you don't have any OpenPAI account and want to deploy an OpenPAI cluster. In pai mode, your trial program will run in pai's container created by Docker. 
```

Comment:
How about change the 1st sentence to:
Starting from v0.2.0, NNI supports one more Training Service Mode: pai mode, which enabled user to run an experiment on [OpenPAI](https://github.com/Microsoft/pai).